teams/teams-ps/teams/New-CsOnlineDirectRoutingTelephoneNumberUploadOrder.md

```diff
@@ -0,0 +1,127 @@
+---
```

Comment:
Typos in this file:

Micorosoft -> Microsoft
asynchronus -> asynchronous
reason of this upload order -> reason for this upload order

Reply:
fixed